src/common/utils/oidc/helper.go

```diff
@@ -280,9 +283,11 @@ func mergeUserInfo(remote, local *UserInfo) *UserInfo {
 	}
```

Comment:
Suggest to put the code these code logic  
https://github.com/reasonerjt/harbor/blob/f4ff369ed0f0d8118100ae54fbc6dc667588ec1e/src/common/utils/oidc/helper.go#L261-L272
into mergeUserInfo function

Reply:
See my comment in the thread:
https://github.com/goharbor/harbor/pull/13312#discussion_r509008990

This is not necessary.